### PR TITLE
NMS-12893: Additions to Application Perspective Monitoring docs

### PR DESCRIPTION
**Pull request by Bonrob2**
### All Contributors

* [ x] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [ x] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [X ] Have you made a comment in that issue which points back to this PR?
* [ X] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [ ] If this is a new or updated feature, is there documentation for the new behavior?
* [ ] If this is new code, are there unit and/or integration tests?
* [ ] If this PR targets a `foundation-*` branch, does it avoid changing files in `$OPENNMS_HOME/etc/`?

### Pull Request Process

One or more reviewers should be assigned to each PR.

If you know that a particular person is subject matter expert in the area your PR affects, feel free to assign one or more reviewers when you create this PR, otherwise reviewers will be assigned for you.

If you have made additions or changes to the documentation, or if you _need_ documentation for these code changes, please make sure a technical writer has looked it over.

Once the reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the Order of the Green Polo) you are welcome to merge the PR.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-12893

### CODE REVIEW

**Comment by Bonrob2** [comment]
Updates to the documentation. NMS-12893

File: opennms-doc/guide-admin/src/asciidoc/text/application-perspective-monitoring/application-perspective-monitoring.adoc [hunk 1]
```diff
@@ -2,29 +2,39 @@
...
+NOTE: Use the `perspective-only` flag in the package definition to define packages that will be used *only* for Application Perspective Monitoring.
 
-When a service is not responsive an outage will be generated with the corresponding perspective.
-To configure _Application Perspective Monitoring_ you must first create an application.
+==== Configuring Application Perspective Monitoring
+
+Application Perspective Monitoring requires at least one {opennms-product-name} Minion on your network.
+Refer to the xref:https://docs.opennms.org/opennms/releases/latest/guide-install/guide-install.html#_monitor_isolated_location_with_minion[Installation Guide] for more information. 
+
+To configure Application Perspective Monitoring, create an  application and then associate a set of services and perspectives from which to monitor those services with the application.
```
Comment (christianpape) [inline opennms-doc/guide-admin/src/asciidoc/text/application-perspective-monitoring/application-perspective-monitoring.adoc:23]: double space: an  application

**Review by christianpape — APPROVED** [review]
Looks good to me.

**Review by fooker — APPROVED** [review]

File: opennms-doc/guide-admin/src/asciidoc/text/application-perspective-monitoring/application-perspective-monitoring.adoc (hunk 1 above)
Comment (fooker) [reply opennms-doc/guide-admin/src/asciidoc/text/application-perspective-monitoring/application-perspective-monitoring.adoc]: Fixed.

**Pull request merged**